README.md

```diff
@@ -0,0 +1,101 @@
+# HTML & CSS3 & JavaScript Course
```

Comment:
- Fill your ``README`` File with your information and project information. You can have a [template](https://github.com/microverseinc/readme-template) provided by `Microverse` that you can use it easily, just change the information.